packages/http-client-python/emitter/src/lib.ts

```diff
@@ -1,54 +1,110 @@
-import { SdkContext, SdkServiceOperation } from "@azure-tools/typespec-client-generator-core";
+import {
+  SdkContext,
+  SdkEmitterOptions,
+  SdkEmitterOptionsSchema,
+} from "@azure-tools/typespec-client-generator-core";
 import { createTypeSpecLibrary, JSONSchemaType, paramMessage } from "@typespec/compiler";
 
-export interface PythonEmitterOptions {
+export interface PythonEmitterOptions extends SdkEmitterOptions {
   "package-version"?: string;
   "package-name"?: string;
-  "output-dir"?: string;
   "generate-packaging-files"?: boolean;
   "packaging-files-dir"?: string;
   "packaging-files-config"?: object;
   "package-pprint-name"?: string;
   "head-as-boolean"?: boolean;
   "models-mode"?: string;
-  tracing?: boolean;
   "company-name"?: string;
   "generate-test"?: boolean;
-  debug?: boolean;
   flavor?: "azure";
-  "examples-dir"?: string;
-  // If true, package namespace will respect the typespec namespace. Otherwise,
-  // package namespace is always aligned with package name.
   "enable-typespec-namespace"?: boolean;
   "use-pyodide"?: boolean;
 }
 
-export interface PythonSdkContext<TServiceOperation extends SdkServiceOperation>
-  extends SdkContext<PythonEmitterOptions, TServiceOperation> {
+export interface PythonSdkContext extends SdkContext<PythonEmitterOptions> {
   __endpointPathParameters: Record<string, any>[];
 }
 
-const EmitterOptionsSchema: JSONSchemaType<PythonEmitterOptions> = {
+export const PythonEmitterOptionsSchema: JSONSchemaType<PythonEmitterOptions> = {
   type: "object",
   additionalProperties: true,
```

Comment:
for azure emitter, we will have another option type and option schema type. so, we could set it to false.

Reply:
oh, since we use the same emitter to test azure and unbranded case, i must set this to true.

Reply:
follow up pr for azure emitter: https://github.com/Azure/autorest.python/pull/3060